routers/api/v1/repo/issue_comment.go

```diff
@@ -508,7 +508,7 @@ func deleteIssueComment(ctx *context.APIContext) {
 		return
 	}
 
-	if err = comment_service.DeleteComment(comment, ctx.User); err != nil {
+	if err = comment_service.DeleteComment(ctx.User, comment); err != nil {
```

Comment:
Why we need to adjust the sequence of the two parameters?

Reply:
As I wrote per discord...: to make new backports less painles, I let go refactor in ...

Since you had concerns I'll remove it

Reply:
@lunny a